content/en/blog/_posts/2022-08-15-local-storage-capacity-isolation-ga.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: blog
+title: "Kubernetes 1.25: Local Storage Capacity Isolation Reaches GA"
+date: 2022-08-15
```

Comment:
Change to 2022-09-19 which is the publish date

Reply:
```suggestion
date: 2022-09-19
```
(and also please change the path of the file to match)

Reply:
@xing-yang Are you active for this issue?

Reply:
Yes.  @jingxu97 Can you please address the review comments?

Reply:
done, also changed the file name.